src/tests/ftest/util/general_utils.py

```diff
@@ -990,7 +990,7 @@ def convert_string(item, separator=","):
     return item
 
 
-def create_directory(hosts, directory, timeout=10, verbose=True,
+def create_directory(hosts, directory, timeout=15, verbose=True,
                      raise_exception=True, sudo=False):
     """Create the specified directory on the specified hosts.
```

Comment:
Minor: timeout description should be updated as well below to reflect 15 seconds now instead of 10 seconds.